azure/test/client.test.js

```diff
@@ -154,6 +154,13 @@ describe('Azure Log Monitoring', function() {
       testHandleLogs(record, expected, true)
     });
 
+    it('should handle byte array properly', function() {
```

Comment:
Can you add a test for the case where the message does not have `records` as well as for a buffer array of string logs?